lib/feature.rb

```diff
@@ -0,0 +1,16 @@
+require './lib/place_order'
+require './lib/menu'
+my_menu = Menu.new
+
```

Comment:
Really nicely defined methods to make clear their function. Really great! Helps make your code cleaner by not using comments.